_drafts/the-drupalorg-complexity.md

```diff
@@ -209,4 +209,4 @@ It would be important to phase in this new feature rather than make a wholesale
 
 We need to be flexible. Github is the future of open source distribution and collaboration—likely for the foreseeable future. However, not every major project is on Github. Linux is on Git repositories with rudimentary tools and a patch-based workflow that relies heavily on email. It works for them. Wordpress is still on Subversion—even though they've started to accept some pull requests on Github. These projects are poised to make the right decision rather than a rash decision.
 
-The sky will not fall if we keep our current model, but we are losing opportunities to grow as a community of contributors.
+The sky will not fall if we keep our current model, but we are losing opportunities to grow as a community of contributors. Rather than a wholesale migration, we must understand the value and history of this web of integration points. Targeting our efforts on specific integrations points can achieve our goal of opening our doors to the developers who live and breathe GitHub, without losing the character of our collaboration. And in the long run, this focus on services and integrations can make us more adaptable to the next change in the broader development landscape.
```

Comment:
Thanks, @hestenet. This is closer to what I intended and definitely closes on the solution.